apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -244,22 +245,35 @@ function initializeThermometer(board) {
   });
 }
 
-function initializeAccelerometer(board) {
+function initializeAccelerometer(board, expressBoard) {
   const accelerometer = new five.Accelerometer({
     board,
     controller: PlaygroundIO.Accelerometer
   });
   accelerometer.start = function() {
     accelerometer.io.sysexCommand([CP_COMMAND, CP_ACCEL_STREAM_ON]);
   };
-  accelerometer.getOrientation = function(orientationType) {
+  accelerometer.getOrientation = function(
+    orientationType,
+    express = expressBoard
+  ) {
     if (undefined === orientationType) {
       return [
         accelerometer.getOrientation('x'),
         accelerometer.getOrientation('y'),
```

Comment:
I think these need to be swapped, in the express case?

Reply:
Re-upping.

Reply:
Followed up offline. Inclination/pitch/roll will be consistent across boards, but @epeach makes a good point that standardizing x/y/z isn't possible because the markings on each board differ.

One option would be to expose acceleration in terms of up-down/left-right/forward-back when holding the board the "standard" way (with the USB cable at the top) but that's less real-world for students.

Reply:
Still have a sneaking concern about tilt axes being standardized between boards but not force axes. Happy to disagree and commit as long as the discrepancy is documented!